lib/libc/stdlib/malloc/malloc_revoke.3

```diff
@@ -29,38 +34,96 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd May 14, 2021
-.Dt MALLOC_REVOKE 3
```

Comment:
Hmm, should have noticed this earlier.  Probably you want to revert back to `MALLOC_REVOKE 3` for the manpage title?  Normally this would match the filename and I suspect in this case you'd rather just keep the current title than renaming the file and adding another MLINK, etc.

Reply:
Oops, that was an artifact of consolidating the two manpages.